### PR TITLE
Fix ic uninitialized warning in release build

### PR DESCRIPTION
**Pull request by jiaqizho**
### Change logs

- Fix ic uninitialized warning in release

### Why are the changes needed?

- Block the release build

### Does this PR introduce any user-facing change?

- nil

### How was this patch tested?

- no spec

### Contributor's Checklist

Here are some reminders and checklists before/when submitting your pull request, please check them:

- [ ] Make sure your Pull Request has a clear title and commit message. You can take [git-commit](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) template as a reference.
- [ ] Sign the Contributor License Agreement as prompted for your first-time contribution.
- [ ] List your communication in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (if has or needed).
- [ ] Document changes.
- [ ] Add tests for the change
- [ ] Pass `make installcheck`
- [ ] Pass `make -C src/test installcheck-cbdb-parallel`
- [ ] Feel free to @cloudberrydb/dev team for review and approval when your PR is ready🥳

### CODE REVIEW

**Comment by jiaqizho** [comment]
@my-ship-it after sign CLA, the CLA bot become reviewer ? so weird about this action 😆 

image

**Comment by tuhaihe** [comment]
Hi, @jiaqizho: it should happen like this and not affect your PR review. Two reviewers with write access are still required for your PR approval.

**Review by yjhjstz — APPROVED** [review]

**Review by my-ship-it — APPROVED** [review]

**Pull request merged**
